chapters/arrays.tex

```diff
@@ -1417,12 +1417,12 @@ \section{Empty Arrays}\label{empty-arrays}
 
 Empty matrices can be constructed with the fill function.  For example:
 \begin{lstlisting}[language=modelica]
-Real A[:,:] = fill(0.0, 0, 1); // a Real 0 x 1 matrix
-Boolean B[:, :, :] = fill(false, 0, 1, 0); // a Boolean 0 x 1 x 0 matrix
+Real A[:,:] = fill(0.0, 0, 1); // an empty Real 0 x 1 matrix
+Boolean B[:, :, :] = fill(false, 0, 1, 0); // an empty Boolean 0 x 1 x 0 matrix
 \end{lstlisting}
 
-It is not possible to access an element of an empty matrix, e.g.\ \lstinline!v[j, k]! cannot be evaluated if \lstinline!v = []! because the assertion fails
-that the index must be bigger than one.
+It is not possible to access the elements of an empty matrix, e.g.\ \lstinline!v[j, k]! cannot be evaluated if \lstinline!v = []! because the assertion fails
```

Comment:
I actually prefer 'an element' to 'the elements'. The reason is that `v[:,:]` can be seen as accessing the elements of the matrix, but `v[j,k]` for scalar `j` and `k` accesses _an_ element.
```suggestion
It is not possible to access an element of an empty matrix, e.g.\ \lstinline!v[j, k]! for scalars \lstinline!j! and \lstinline!k! cannot be evaluated if \lstinline!v = []! because the assertion fails
```

Reply:
It should also be merged with the next line because the next line is part of the same sentence. Also "the assertion fails that..." is not quite correct. It is better to say "the assertion that... fails."

Reply:
Note my suggestion to replace this sentence with more useful examples is taking care of this.

Reply:
Yes, there is also @HansOlsson's suggestion above which does not correct the line break and the weird phrasing. So I was not sure which of the suggestions is considered as the main candidate.